docs/fuzz.md

```diff
@@ -1,43 +1,40 @@
 # Fuzzing LND #
 
-The `fuzz` package is organized into subpackages which are named after the `lnd` package they test. Each subpackage has its own set of fuzz targets.
-
-## Setup and Installation ##
-This section will cover setup and installation of the fuzzing binaries.
-
-* The following is a command to build all fuzzing harnesses:
+The following runs all fuzz tests on default settings:
 ```shell
-$  make fuzz-build
+$  make fuzz
 ```
-
-* This may take a while since this will create zip files associated with each fuzzing target.
-
-* The following is a command to run all fuzzing harnesses for 30 seconds:
+The following runs all fuzz tests inside the lnwire package, each for a total of 1 minute, using 4 procs. 
```

Comment:
According to the `fuzz_flags.mk` this is `30s` instead of 1 minute.

Reply:
The below example manually overrides `fuzztime` to `1m`.